src/views/LoanDetails/LoanDetails.test.js

```diff
@@ -265,228 +265,278 @@ const propsData = {
 
 const renderLoanProxyDetails = (props1) => renderWithRouter(<LoanDetails {...props1} />);
 
-describe('Render LoanProxyDetails component', () => {
-  it('When props ID and proxy ID are same with Loan Missing', () => {
-    mockGetLodash.mockReset();
-    mockGetLodash.mockImplementation(() => {
-      return 'Lost and paid';
+describe('LoanDetails', () => {
+  describe('Render LoanProxyDetails component', () => {
+    it('When props ID and proxy ID are same with Loan Missing', () => {
+      mockGetLodash.mockReset();
+      mockGetLodash.mockImplementation(() => {
+        return 'Lost and paid';
+      });
+      renderLoanProxyDetails({
+        ...propsData,
+        loanIsMissing: true,
+      });
+      expect(screen.getAllByText('ui-users.loan404')).toBeTruthy();
     });
-    renderLoanProxyDetails({
-      ...propsData,
-      loanIsMissing: true,
+    it('When props ID and proxy ID are same', () => {
+      mockGetLodash.mockReset();
+      mockGetLodash.mockImplementation(() => {
+        return 'Lost and paid';
+      });
+
+      const { getByText } = renderLoanProxyDetails({
+        ...propsData,
+        loanIsMissing: false,
+      });
+      userEvent.click(getByText('stripes-smart-components.cddd.changeDueDate'));
+      userEvent.click(getByText('ui-users.renew'));
+      userEvent.click(getByText('ui-users.loans.claimReturned'));
+      expect(screen.getAllByText('ui-users.loans.claimReturned')).toBeTruthy();
     });
-    expect(screen.getAllByText('ui-users.loan404')).toBeTruthy();
-  });
-  it('When props ID and proxy ID are same', () => {
-    mockGetLodash.mockReset();
-    mockGetLodash.mockImplementation(() => {
-      return 'Lost and paid';
+    it('When props ID and proxy ID are same with data', () => {
+      mockGetLodash.mockReset();
+      mockGetLodash.mockImplementation(() => {
+        return '';
+      });
+      renderLoanProxyDetails({
+        ...propsData,
+        loanIsMissing: false,
+      });
+      expect(screen.getAllByText('ui-users.loans.claimReturned')).toBeTruthy();
     });
+    it('When props ID and proxy ID are same with claims', () => {
+      mockGetLodash.mockReset();
+      mockGetLodash.mockImplementation((item, item2) => {
+        return item2.includes('contributors') ? undefined : 'Claimed returned';
+      });
+      const { getByText, getAllByText } = renderLoanProxyDetails(propsData);
 
-    const { getByText } = renderLoanProxyDetails({
-      ...propsData,
-      loanIsMissing: false,
+      userEvent.click(getAllByText('ui-users.loans.declareLost')[0]);
+      userEvent.click(getAllByText('ui-users.loans.declareLost')[1]);
+      userEvent.click(screen.queryAllByRole('button')[0]);
```

Comment:
I guess it's better to use one approach to be consistent here - either use screen.someQuery or distract all queries from your render component method

Reply:
@mariia-aloshyna 
It is an existing code. It can be addressed in the scope of https://issues.folio.org/browse/UIU-3001
I have added a reference to your comment in the ticket description.